web/src/lib/pagination.ts

```diff
@@ -1,3 +1,107 @@
+const DEFAULT_PAGE_SIZE = 10;
+
+/**
+ * Helper function returns the url of the updated param page
+ *
+ * @param {string} currentPageUrl - The URL of the current page
+ * @param {boolean} updatedPage - The page number we would like to update to
```

Comment:
1. This should be number
2. I don't think it's worth maintaining the JSDoc types for every function